jupyter/amd/pytorch/ubi9-python-3.9/Pipfile

```
@@ -0,0 +1,64 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[[source]]
+url = "https://download.pytorch.org/whl/nightly/rocm6.1/"
```

I found the version on this [link](https://pytorch.org/get-started/previous-versions/#linux-and-windows) to be 6.0. Does this update automatically somehow?

```suggestion
url = "https://download.pytorch.org/whl/nightly/rocm6.0/"
```